fire/cli/mtl.py

```diff
@@ -841,3 +908,356 @@ def regn(projektnavn: str, **kwargs) -> None:
+            fire.cli.print(
+                f"ADVARSEL: Nyoprettet punkt index {i} har ingen gyldig afmærkning anført",
+                fg="red",
+                bg="white",
+                bold=True,
+            )
+        pi_a = PunktInformation(infotype=afmærkning_pit, punkt=nyt)
+        nyt.punktinformationer.append(pi_a)
+
+        # Tilføj punktbeskrivelsen som punktinformation, hvis anført
+        if not pd.isna(nyetablerede["Beskrivelse"][i]):
+            pi_b = PunktInformation(
+                infotype=beskrivelse_pit,
+                punkt=nyt,
+                tekst=nyetablerede["Beskrivelse"][i],
+            )
+            nyt.punktinformationer.append(pi_b)
+
+        genererede_punkter[i] = nyt
+
+    if len(genererede_punkter) == 0:
+        fire.cli.print("Ingen nyetablerede punkter at registrere")
+        return
+
+    # Gør klar til at persistere
+
+    # Generer sagsevent
+    sagsevent = Sagsevent(sag=sag)
+    sagsevent.id = str(uuid4())
+    er = "er" if len(genererede_landsnumre) > 1 else ""
+    sagseventtekst = f"Oprettelse af punkt{er} {', '.join(genererede_landsnumre)}"
+    sagseventinfo = SagseventInfo(beskrivelse=sagseventtekst)
+    sagsevent.sagseventinfos.append(sagseventinfo)
+
+    # Generer dokumentation til fanebladet "Sagsgang"
+    sagsgangslinje = {}
+    sagsgangslinje["Dato"] = pd.Timestamp.now()
+    sagsgangslinje["Initialer"] = initialer
+    sagsgangslinje["Hændelse"] = "punktoprettelse"
+    sagsgangslinje["Tekst"] = sagseventtekst
+    sagsgangslinje["uuid"] = sagsevent.id
+    sagsgang = sagsgang.append(sagsgangslinje, ignore_index=True)
+
+    # Persister punkterne til databasen
+    fire.cli.print(sagseventtekst, fg="yellow", bold=True)
+    if "ja" != input(
+        "-->  HELT sikker på at du vil skrive punkterne til databasen (ja/nej)? "
+    ):
+        fire.cli.print("Dropper skrivning")
+        return
+    firedb.indset_flere_punkter(sagsevent, list(genererede_punkter.values()))
+
+    # ... og marker i regnearket at det er sket
+    for k in genererede_punkter:
+        nyetablerede.at[k, "uuid"] = genererede_punkter[k].id
+
+    # Skriv resultater til resultatregneark
+    resultater["Sagsgang"] = sagsgang
+    # Drop numerisk index
+    nyetablerede = nyetablerede.reset_index(drop=True)
+    resultater["Nyetablerede punkter"] = nyetablerede
+    skriv_resultater(projektnavn, resultater)
+
+    fire.cli.print(
+        f"Punkter oprettet. Kopiér nu faneblade fra '{projektnavn}-resultat.xlsx' til '{projektnavn}.xlsx'"
+    )
+
+
+# ------------------------------------------------------------------------------
+# Her starter sagsoprettelsesprogrammet...
+# ------------------------------------------------------------------------------
+@mtl.command()
+@fire.cli.default_options()
+@click.argument(
+    "projektnavn", nargs=1, type=str,
+)
+@click.argument(
+    "initialer", nargs=1, type=str,
+)
+def opret_sag(projektnavn: str, initialer: str, **kwargs) -> None:
```

Comment:
Jeg mener det her bør bo under `fire sag [opret|åbn|luk]`, det er heldigvis nemt at flytte

Reply:
Det er rigtigt: Vi mangler en "opret sag" funktionalitet under `fire sag`. Her bliver det lidt mere omfattende, fordi `opret sag` under `mtl`også skal foretage de indledende armbøjninger mht. at gøre regneark klar til brug. Jeg tænker at lade det hvile lidt, indtil vi har mindst to anvendelser af "opret sag" funktionaliteten, og så refaktorisere når det bliver lidt mere tydeligt hvor snittet bør ligge (jf issue #206)